src/test/resources/docker/initialize/data/server-configs.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "id": "Configuration/Custom/Configuration2139581",
+    "type":"servicenow.Server",
+    "title":"servicenow",
+    "url":"https://servicenowcd.service-now.com/",
+    "username":"joris",
+    "password":"joris",
```

Comment:
i wonder if the user/password should be left in there..  maybe one can use their own account.